packages/wmr/src/lib/acorn-traverse.js

```diff
@@ -489,7 +489,7 @@ const TYPES = {
 							// We can drop the whole matched string if it only contains
 							// whitespace characters, because that means we're at the
 							// beginning or at the end of the JSXText node
-							const replacement = /\w/.test(value) ? ' ' : '';
+							const replacement = /[.\S]/.test(value) ? ' ' : '';
```

Comment:
Good catch about inverting the rege 👍 Quick question regarding including `.`: Did you found a case where this signals a newline?

Reply:
Nah, this was all you mate!

Regarding `.` matching newlines, the dot notation says: "any character _except_ newline". So, that means also whitespace, and that's why I added the `\S`(not whitespace) in the range so we only get matching any chars expect: newlines and whitespaces.
Did I get it right?

Reply:
> Regarding `.` matching newlines, the dot notation says: "any character _except_ newline".

Shouldn't that be `[^\S]` instead of `[.\S]`?

Reply:
Oh, I understand the confusion now. `\S` capital 'S' means _not_ whitespace, lowercase 's' means whitespace.

Reply:
`[.]` matches a literal `.` character (this one always throws me haha).

I think `[^\s]` works here:

```js
/[^\s]/.test(' ') // false
/[^\s]/.test('\n') // false
/[^\s]/.test(' \n \t') // false
/[^\s]/.test('a') // true
/[^\s]/.test('\na ')
```

Reply:
Uh oh! 😲  What in the name of holly regex rules... TIL. Then yup! That's what it should be. Changing it.